tests/testapp/settings.py

```diff
@@ -12,10 +12,11 @@
 import os
 from pathlib import Path
 
+from pictures import conf
```

Comment:
Please just set the queue name, but don't import the `conf` module here. It might lead to some strange recursive imports. You can just hardcode the queue name in the settings below.

Reply:
Make sense